app/tasks/maintenance_tasks/csv_task.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module MaintenanceTasks
+  # A mixin that provides a Task with the ability to process CSV files.
+  module CsvTask
```

Comment:
It seems to me that this module doesn't need to exist and we can default `collection` and `count` to this behavior if `csv_content` is present in the task. The content is only present if the user upload to the run right?

Why make users have to add this module if the system can do everything automatically?